File: single-language-analysis/dataset-files/java/TestSmell069.java
// Original URL: https://github.com/skolanko222/CLOBprocessor/blob/19097d5f286ba78e3431883826c0ef734facdd53/src/ClobProcessorTest.java#L86-L95

public class TestSmell069 {
    @Test
    public void testSearchFreeTextDocument() throws SQLException, InterruptedException {
        String content = "Safe searchable content.";
        clobProcessor.saveDocument(null, content);
        //wait for the document to be indexed
        Thread.sleep(5000);
        ArrayList<String> results = clobProcessor.searchFreeTextDocument("contents");
        System.out.println(results);
        assertFalse(results.isEmpty());
    }
}

File: single-language-analysis/dataset-files/python/test_smell_069.py
import pytest

class TestClobProcessor:
    def test_smell_069(self):
        content = "Safe searchable content."
        clob_processor.save_document(None, content)
        # wait for the document to be indexed
        time.sleep(5)
        results = clob_processor.search_free_text_document("contents")
        print(results)
        assert len(results) > 0